admin/google_search_console/class-gsc-category-filters.php

```diff
@@ -0,0 +1,179 @@
+	}
+
+	/**
+	 * Setting the view counts based on the saved data. The info will be used to display the category filters
+	 *
+	 * @param string $platform
+	 */
+	private function set_counts( $platform ) {
+		$platform_counts = $this->get_counts();
+
+		if ( array_key_exists( $platform, $platform_counts ) ) {
+			$this->category_counts = $this->parse_counts( $platform_counts[ $platform ] );
+		}
+
+		$this->set_filter_values();
+	}
+
+	/**
+	 * Getting the options with the counts
+	 *
+	 * @return mixed
+	 */
+	private function get_counts() {
+		return get_option( WPSEO_GSC_Count::OPTION_CI_COUNTS, array() );
+	}
+
+	/**
+	 * Setting the values for the filter
+	 */
+	private function set_filter_values() {
+		$this->set_filter_value( 'access_denied', __( 'Access denied', 'wordpress-seo' ), __( 'Server requires authentication or is blocking Googlebot from accessing the site.', 'wordpress-seo' ) );
+		$this->set_filter_value( 'faulty_redirects', __( 'Faulty redirects', 'wordpress-seo' ) );
+		$this->set_filter_value( 'not_followed',__( 'Not followed', 'wordpress-seo' ) );
+		$this->set_filter_value( 'not_found', __( 'Not found', 'wordpress-seo' ), __( 'URL points to a non-existent page.', 'wordpress-seo' ) );
+		$this->set_filter_value( 'other', __( 'Other', 'wordpress-seo' ), __( 'Google was unable to crawl this URL due to an undetermined issue.', 'wordpress-seo' ) );
+		/* Translators: %1$s: expands to '<code>robots.txt</code>'. */
+		$this->set_filter_value( 'roboted', __( 'Blocked', 'wordpress-seo' ), sprintf( __( 'Googlebot could access your site, but certain URLs are blocked for Googlebot in your %1$s file. This block could either be for all Googlebots or even specifically for Googlebot-mobile.', 'wordpress-seo' ), '<code>robots.txt</code>' ) );
+		$this->set_filter_value( 'server_error', __( 'Server Error', 'wordpress-seo' ), __( 'Request timed out or site is blocking Google.', 'wordpress_seo' ) );
+		$this->set_filter_value( 'soft_404', __( 'Soft 404', 'wordpress-seo' ), __( "The target URL doesn't exist, but your server is not returning a 404 (file not found) error.", 'wordpress-seo' ) );
+	}
+
+	/**
+	 * Add new filter value to the filter_values
+	 * @param string $key
+	 * @param string $value
+	 * @param string $description
+	 */
+	private function set_filter_value( $key, $value, $description = '' ) {
+		$this->filter_values[ $key ] = array(
+			'value'       => $value,
+			'description' => $description,
+		);
+	}
+
+	/**
+	 * Creates a filter link
+	 *
+	 * @param string  $key
+	 * @param integer $count
+	 *
+	 * @return string
+	 */
+	private function create_view_link( $key, $count ) {
+		$href  = add_query_arg( array( 'category' => $key, 'paged' => 1 ) );
+
+		$class = 'gsc_category';
+		if ( $this->current_view == $key ) {
+			$class .= ' current';
+		}
+
+		$title = '';
+		if ( $this->filter_values[ $key ]['description'] !== '' ) {
+			$title = " title='" . esc_attr( $this->filter_values[ $key ]['description'] ) . "'";
```

Comment:
You shouldn't use a title attribute because of accessibility. Especially if it's essential information such as the description. Title attributes are supported variably in screenreaders and are completely inaccessible on touch devices.